app/views/partials/contact_hmrc_form.scala.html

```diff
@@ -24,8 +24,12 @@
     <header class="page-header">
         <h1>@Messages("contact.heading")</h1>
     </header>
-    <p>@Html(Messages("contact.form.specific.query", uk.gov.hmrc.urls.Link.toExternalPage(url = "http://search2.hmrc.gov.uk/kb5/hmrc/contactus/home.page", value = Some("contact HMRC"), dataAttributes = Some(Map
-            ("journey-click" -> "contact-page:Click:Contact HMRC"))).toHtml))</p>
+    <p>@Html(
+        Messages(
+            "contact.form.specific.query",
+            <a href="http://search2.hmrc.gov.uk/kb5/hmrc/contactus/home.page" target="_blank" data-sso="false" data-journey-click="contact-page:Click:Contact HMRC" rel="external noopener noreferrer">contact HMRC<span class="visuallyhidden"> (opens in new tab)</span></a>
```

Comment:
this LGTM, but I think that (pre-existing) "contact HMRC" should be using `Messages("contact.hmrc.link")` instead.

Reply:
@ellamdav Good spot, change made.